decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
```

Comment:
- Hey, let's kindly add the method to access the nameable to give the read/write access to the name. 
`attr_accessor :nameable`

Reply:
OK, it's done! thanks!